src/components/cl/basic/cl_basic.h

```diff
@@ -15,7 +15,7 @@
 #define UCC_CL_BASIC_DEFAULT_SCORE 10
 #endif
 
-#define UCC_CL_BASIC_NUM_TLS 2
```

Comment:
not needed. UCC_CL_BASIC_NUM_TLS is no longer used - must be removed from master

Reply:
fixed, I think I just removed it - although we should probably remove it as a separate PR wdyt?